src/components/PostComment.jsx

```diff
@@ -0,0 +1,27 @@
+import { useState } from "react";
+import { postCommentToDataBase } from "../api";
+
+export default function PostComment({ article_id }) {
+  const [commentInput, setCommentInput] = useState({});
+  const [message, setMessage] = useState("");
+
+  function handlesChange(event) {
+    setCommentInput({ user: "tickle122", body: event.target.value });
```

Comment:
Personally, I would save tickle122 as a variable called user higher up and pass it in here. The code will work exactly the same, but it will indicate to anyone viewing your code that you're aware that this typically wouldn't be hardcoded.